Makefile

```diff
@@ -147,6 +147,18 @@ self-signed-https-certs: ## Generates self-signed certs for TESTING the HTTPS co
 vagrant-package: ## Package up a vagrant box of the last stable SD release
 	@devops/scripts/vagrant_package.sh
 
+.PHONY: clean
+clean: ## DANGER! Purges all site-specific info and developer files from project.
```

Comment:
what about adding a wee confirmation prompt (something like [this](https://stackoverflow.com/questions/47837071/making-make-clean-ask-for-confirmation)) such that we minimize chance of someone accidentally running `make clean` not understanding what they are doing? Admittedly unlikely but since it's a simple change it's probably worthwhile